src/Data/Higher/Graph.hs

```diff
@@ -125,7 +125,7 @@ instance HEqF f => Eq (Rec f (Const Int) a)
   where a == b = eqRec 0 a b
 
 instance HShowF f => Show (Graph f a)
-  where showsPrec n = showsPrec n . (unGraph :: Graph f ~> Rec f (Const Char))
+  where showsPrec n (Graph rec) = showsPrec n (rec :: Rec f (Const Char) a)
```

Comment:
I had forgotten that the type annotation would be necessary due to unwrapping the `Graph`! Good fix 👍🏻